REUSE.toml

```diff
@@ -25,6 +25,11 @@ precedence = "aggregate"
 SPDX-FileCopyrightText = "The PyPSA-Earth and PyPSA-Eur Authors"
 SPDX-License-Identifier = "CC-BY-4.0"
 
+[[annotations]]
+path = "test/utils/agg_p_nom_minmax.csv"
+precedence = "aggregate"
+SPDX-FileCopyrightText = "The PyPSA-Earth and PyPSA-Eur Authors"
+SPDX-License-Identifier = "CC-BY-4.0"
```

Comment:
Thanks for keeping REUSE in sight. Just putting a reminder here that if we'll decide to add `agg_p_nom_minmax` also to `data` would be good to list it here as well